main.yml

```diff
@@ -228,6 +228,21 @@ rhel:
       - id: remediations
   top_level: true
 
+ruledev:
+  title: Rule Development
+  channel: '#flip-mode-squad'
+  deployment_repo: https://github.com/RedHatInsights/ruledev-frontend-build
+  frontend:
+    title: Rule Development
+    paths:
+      - /ruledev
```

Comment:
This should be the full path to the app. If we're putting it in the Staging bundle, that's `/staging/ruledev`.